src/main/java/com/microsoft/jenkins/kubernetes/wrapper/KubernetesClientWrapper.java

```diff
@@ -321,16 +321,16 @@ public void createOrReplaceSecrets(
         log(Messages.KubernetesClientWrapper_prepareSecretsWithName(secretName));
 
         DockerConfigBuilder dockerConfigBuilder = new DockerConfigBuilder(credentials);
-        String dockercfg = dockerConfigBuilder.buildDockercfgBase64();
+        String dockercfg = dockerConfigBuilder.buildDockercfgString();
```

Comment:
Why do we need this change?

Reply:
I think the official kubernetes sdk will automatically perform base64 encode, using the original string can create a secret, and using base64 data will result in
```
 "message": "Secret \"test\" is invalid: data[.dockercfg]: Invalid value: \"\u003csecret contents redacted\u003e\": invalid character 'e' looking for beginning of value",
```